helpers/reference-data-reader/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { ethers } from 'ethers'
+
+export type ReferenceDataPrice = (
+  contractAddress: string,
+  multiply: number,
+  meta?: Record<string, unknown>,
+) => Promise<number>
+
+export const getLatestAnswer: ReferenceDataPrice = async (
```

Comment:
We already have a package like this :) We should just use [`@chainlink/contracts`](https://www.npmjs.com/package/@chainlink/contracts) where `ethers.Contract` with all the types should be exposed.

Even if we want to have additional code (utils or best practices) for accessing contracts, that should also probably be added to `@chainlink/contracts` or a subpackage to that like `@chainlink/contracts-utils`.

Reply:
I'm not sure if I follow. This package has been published as `@chainlink/reference-data-reader`. Do you want us to use a different name, or move it to a different repo?

Reply:
At this point, I don't see the need for a separate package as it does the same, generates `ethers.Contract` from ABI, as `@chainlink/contracts` NPM package we already have. So my suggestion is to just include and use that.

Reply:
```javascript
import { AggregatorInterface } from '@chainlink/contracts/ethers/v0.6/AggregatorInterface'
// TODO: add provider
const answ =  await aggregator.latestAnswer()
```

Reply:
If you have additional plans for the `reference-data-reader` package, that will justify the need for a separate package, then ok. Just seemed like an overkill for these few lines of code, for now.